pkg/util/exec/exec.go

```diff
@@ -117,25 +118,17 @@ func Run(cmd *exec.Cmd, streams ioutil.StdStreams) error {
 }
 
 func terminate(cmd *exec.Cmd, errCh <-chan error) {
```

Comment:
I guess we don't need to pass errCh to this function anymore.

Reply:
yeah thanks for the tips!